storybook/src/components/card/card.core-template.ts

```diff
@@ -21,20 +21,14 @@ export const coreCard: ComponentImplementation<Card<never>> = {
       selectable,
       content,
       interactions,
-      image,
-      imageAlt,
-      imageShape,
-      clickable = false,
       href,
       mode,
       dsoCardClicked,
     }: Card<TemplateResult>) {
       return html`
         <dso-card
-          href=${ifDefined((!clickable && href) || undefined)}
+          href=${ifDefined(href || undefined)}
```

Comment:
Volgens de modellering zal een `href` altijd aanwezig zijn

Reply:
`IfDefined` is weg